json/src/main/java/tech/tablesaw/io/jsonl/JsonlReadOptions.java

```diff
@@ -0,0 +1,186 @@
+package tech.tablesaw.io.jsonl;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.Reader;
+import java.io.StringReader;
+import java.net.URL;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Function;
+
+import tech.tablesaw.api.ColumnType;
+import tech.tablesaw.io.ReadOptions;
+import tech.tablesaw.io.Source;
+
+public class JsonlReadOptions extends ReadOptions {
```

Comment:
Is there anything here that differs from `JsonReadOptions`? I wonder if we might be able to just reuse that existing class vs creating a new one?

Reply:
I think it's a better design to assume that Jsonl might eventually have different options. If you hate that feel free to change it.